core/src/test/java/apoc/load/LoadJsonTest.java

```diff
@@ -321,19 +322,16 @@ public void testLoadJsonWithPathOptions() throws Exception {
                 });
     }
 
-    @Test(expected = QueryExecutionException.class)
-    public void testLoadJsonByUrlInConfigFileWrongKey() throws Exception {
-
-        try {
-            testResult(db, "CALL apoc.load.json($key)",map("key","foo"), (r) -> r.hasNext());
-        } catch (QueryExecutionException e) {
-            Throwable except = ExceptionUtils.getRootCause(e);
-            assertTrue(except instanceof IOException);
-            final String message = except.getMessage();
-            assertTrue(message.startsWith("Cannot open file "));
-            assertTrue(message.endsWith("foo for reading."));
-            throw e;
-        }
+    @Test
+    public void testLoadJsonByUrlInConfigFileWrongKey() {
+        QueryExecutionException e = assertThrows(QueryExecutionException.class,
+                () -> testResult(db, "CALL apoc.load.json($key)",map("key","foo"), (r) -> r.hasNext())
+        );
+        Throwable except = ExceptionUtils.getRootCause(e);
+        assertTrue(except instanceof IOException);
+        final String message = except.getMessage();
+        assertTrue(message.startsWith("Cannot open file "));
```

Comment:
Do we know what is in the middle of this string?

Reply:
It contains the full path which is user/CI dependent (in my case is  /Users/louise/Documents/neo4j-apoc-core/core/) so there is a good reason it doesn't check for the whole message

Reply:
Ah that makes sense :) Thanks